clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchRequest.java

```diff
@@ -71,7 +72,8 @@ public Builder(String groupId,
                        boolean requireStable,
                        List<TopicPartition> partitions,
                        boolean throwOnFetchStableOffsetsUnsupported) {
-            super(ApiKeys.OFFSET_FETCH);
+            // It can only be used with topic names.
+            super(ApiKeys.OFFSET_FETCH, ApiKeys.OFFSET_FETCH.oldestVersion(), (short) 9);
```

Comment:
Since version 9 is the last version that could use topic names, do we need a static variable for it?
For example: `LAST_TOPIC_NAME_SUPPORT_VERSION`